hazelcast/src/main/java/com/hazelcast/sql/impl/client/ReadOptimizedLruCache.java

```diff
@@ -90,6 +91,16 @@ public void put(K key, V value) {
         }
     }
 
+    public V computeIfAbsent(K key, Function<K, V> mappingFn) {
```

Comment:
note that this method behaves in different way that `CHM.computeIfAbsent`. This one is "racy" - may overwrite value in map. That might be sufficient for our use case but has to be documented.

Reply:
True, that's why I asked only for tests, but didn't point on thread-unsafe behavior.

Reply:
d441f957

Reply:
I found it harder to document the behavior than to fix the behavior. However, due to its blocking nature, I will revert it back to the thread-unsafe version, and document the behavior.

Reply:
[`e1c3fd70..b132d684`][1]

[1]: https://github.com/hazelcast/hazelcast/compare/e1c3fd70d93fb0b4956297b6031479d37166855f..b132d684268595d3468902788ffb8a7e401c6df6